pkg/virtctl/ssh/wrapped.go

```diff
@@ -29,13 +27,14 @@ func (o *SSH) runLocalCommandClient(kind, namespace, name string) error {
 
 func (o *SSH) buildProxyCommandOption(kind, namespace, name string) string {
 	proxyCommand := strings.Builder{}
-	proxyCommand.WriteString("-o ProxyCommand=")
+	proxyCommand.WriteString("-o ProxyCommand=\"")
 	proxyCommand.WriteString(os.Args[0])
 	proxyCommand.WriteString(" port-forward --stdio=true ")
 	proxyCommand.WriteString(fmt.Sprintf("%s/%s.%s", kind, name, namespace))
 	proxyCommand.WriteString(" ")
 
 	proxyCommand.WriteString(strconv.Itoa(o.options.SshPort))
+	proxyCommand.WriteString("\"")
```

Comment:
more than unit tests we need to have a functional test as it wasn't working because the server refuses the key for the user `-i` as you can see from the linked issue. I need to give a look if we already have some functional test for virtctl